### DISCUSSION TITLE
Having issues with the reverse proxy sample

### DISCUSSION THREAD

**Discussion by jsm174** [cesanta/mongoose#2453] [Ask questions, get answers] [answered]
I'm trying to use the reverse proxy sample verbatim minus changing the urls:

```
static const char *s_backend_url = "http://192.168.1.170:1234";
static const char *s_listen_url = "http://0.0.0.0:8000";
```

When I hit the url:

http://localhost:8000/index.html

It sometimes makes one successful request. 

Other times it just sits there. It's almost like it isn't closing a connection.

![Screenshot 2023-11-03 at 1 18 36 PM](https://github.com/cesanta/mongoose/assets/1197137/58359af7-170c-48ca-94d4-21c9a339d38f)

If I try to proxy the mongoose.ws website, it does get pretty far, but then eventually locks up:

`static const char *s_backend_url = "http://mongoose.ws";`

![Screenshot 2023-11-03 at 1 25 43 PM](https://github.com/cesanta/mongoose/assets/1197137/32ae0840-998a-426a-9f68-5e0368e46066)

Any ideas what could be up with this? The endpoints I am trying to proxy, are extremely small.

**Comment by scaprile** [comment, marked as answer]
TL;DR
Sorry, I don't have the time to follow your hexdump.
Please put a network sniffer (Wireshark) on each side of the connection and analyze the traffic.
If you have an actual issue, please open an issue, follow the issue template, and attach the capture files, be specific.

**Reply by jsm174** [reply]
I'm sorry. The dump wasn't really as relevant as I thought it would be prior to trying to reverse proxy the mongoose.ws site and getting the same results.
